### DISCUSSION TITLE
Announcement: Eligibility List for POAP Claims and Project Updates

### DISCUSSION THREAD

**Discussion by darwintree** [Conflux-Chain/conflux-documentation#199] [Announcements]
## POAP Eligibility List POAP领取资格名单

We are thrilled to unveil the eligibility list of contributors who are qualified to claim Proof of Attendance Protocol (POAP) tokens. This acknowledgment serves as a testament to your invaluable contributions to our project. We deeply appreciate your support and highly regard the insights and expertise you infuse into our community.
以下为本季度有资格领取 POAP 的贡献者白名单。非常感谢名单中的各位对本项目的支持。

The following Crowdin contributors are eligible:
以下的Crowdin贡献者有资格领取POAP：

- [w33d](https://crowdin.com/profile/w33d) (distinguished contributor)
- [cyclee](https://crowdin.com/profile/cyclee)
- [jasonyou184](https://crowdin.com/profile/jasonyou184)
- [louisl.eth](https://crowdin.com/profile/louisl.eth)
- [effectchen](https://crowdin.com/profile/effectchen)
- [cike](https://crowdin.com/profile/cike)

The following GitHub contributors are eligible:
以下的GitHub贡献者有资格领取POAP：

- [Euroanchor](https://github.com/Euroanchor)
- [atharvadeosthale](https://github.com/atharvadeosthale) 
- [wanliqun](https://github.com/wanliqun) 
- [Pana](https://github.com/Pana)
- [0xn1c0](https://github.com/0xn1c0) 
- [darwintree](https://github.com/darwintree) 

The eligibility list is curated in accordance with our [CONTRIBUTING guideline](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING). If you believe your contribution has been overlooked, please do not hesitate to contact us.
白名单是根据[CONTRIBUTING指导原则](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING)所挑选的。如果你有对应贡献但被遗漏在了名单外，欢迎联系我们。

<details>
<summary>For Newcomers: How to Qualify for Minting the POAP </summary>
To qualify for minting the POAP, your contribution must have been "ACCEPTED", which means:

* Any issue has been created with the "ACCEPTED" label added by an admin
* Any Pull Request has been successfully merged into the main branch
* Any translation string has been thoroughly reviewed

</details>
<details>
<summary>对新参与者的提示：如何有资格铸造POAP</summary>
要想有资格铸造POAP，你的贡献必须被"接受"，即：

* 管理员添加了"接受"标签的任何issue
* 成功合并到主分支的任何Pull Request
* 被彻底审查的任何翻译字符串（Crowdin）

</details>

We acknowledge and reward substantial contributions with special POAPs. For this season, [w33d](https://crowdin.com/profile/w33d) will have the privilege to mint the special POAP for his remarkable contribution to the Simplified Chinese translations.
我们为作出了重大贡献的参与者颁发特殊的POAP。本季度，[w33d](https://crowdin.com/profile/w33d)因其对简体中文翻译的突出贡献，将有权铸造特殊的POAP。

Please be aware that the POAP issuance rules are subject to review and potential modifications every 3 months. This ensures that our guidelines remain pertinent and effective in encouraging and rewarding valuable contributions.
请注意，POAP的发行规则每3个月会进行审查和可能的修改，以确保我们的指导规则仍然可以有效地鼓励和奖励有社区的贡献。

In other developments, we are delighted to announce that the POAP contract and frontend application are currently under development. The contributor POAP will be deployed as a "dual-space-nft" project: a core space ERC721 contract and an eSpace ERC721 contract will be bound and function as if they were a single NFT contract. Moreover, an NFT owner can specify their core/eSpace address as they prefer. We are diligently testing contract logic to ensure security, so we kindly ask for your patience until everything is finalized.
此外，POAP合约和前端应用目前正在开发中。贡献者POAP将被部署为一个"双空间-nft"项目：一个core空间ERC721合约和一个eSpace ERC721合约将被绑定，且作为一个单一的NFT合约运作。NFT所有者可以根据自己的喜好指定他们的core/eSpace地址。我们正在努力测试合约逻辑以确保安全，因此还请耐心等待开发工作完成。
Additionally, we are excited to reveal our POAP design— the former as the general POAP and the latter as the POAP for distinguished contributors.

<img width="200" alt="3_normal" src="https://github.com/Conflux-Chain/conflux-documentation/assets/17946284/40b736d2-a421-4de0-9858-e15b7e7511f9"><img width="200" alt="3_outstanding" src="https://github.com/Conflux-Chain/conflux-documentation/assets/17946284/da60e4cc-bb40-4206-be6c-4eae7d8e9c64">

In terms of appearance, general POAPs are indistinguishable from those issued to distinguished contributors, except for the color scheme. We aim to convey the message that we respect and appreciate every contributor, recognizing that each one plays a crucial role in our project.
在外观上，除了颜色外，POAP与给予杰出贡献者的POAP是完全相同的。我们想表达的是，虽然每一位参与者的投入并不相同，但我们不会因此而轻视任何一位贡献者的付出。感谢每一位贡献者为我们的项目作出的努力，你们每一个人在我们的项目中都扮演着关键的角色。

Thank you for your assistance in enhancing our project. If you have any queries or require assistance, please feel free to create an issue in our Github repository.
感谢各位持续帮助我们改进项目。如果你还有任何疑问或需要帮助，请随时在我们的Github仓库中创建issue。

**Comment by darwintree** [comment]
Congratulations!
@Euroanchor @atharvadeosthale  @wanliqun @Pana @0xn1c0 @darwintree

**Comment by LouisL19851** [comment]
怎么领取？

**Reply by darwintree** [reply]
合约还在测试中。上线前会再进行通知。Github贡献者能够直接通过Github授权进行领取。Crowdin还在集成中。

**Comment by darwintree** [comment]
The Contributor POAP Q2 is now Live!

- Forum link(including minting tutorial): [English version](https://forum.conflux.fun/t/announcing-the-contributor-poap-dualspace-nft-for-q2-2023-now-live/19224) / [Chinese version](https://forum.conflux.fun/t/2023-poap-dualspace-nft/19228)
- The estimated deadline for minting the Q2 NFTs is Sep 29 2023 (Block 192365102)

Thank you for your contribution! Leave your comments here if any issue occurs. @Euroanchor @atharvadeosthale  @wanliqun @Pana

**Comment by LouisL19851** [comment]
试过了，无法领取
![image](https://github.com/Conflux-Chain/conflux-documentation/assets/101488673/122e1bf7-2013-4a74-84dd-2a2437f75439)

**Reply by LouisL19851** [reply]
github和crowdin都试过了，都不行，core和espace也试过

**Reply by darwintree** [reply]
抱歉 这个看起来是白名单写入时的失误。大小写被错误地设置了。可以再次尝试通过crowdin mint吗？
BTW, 你可以通过https://confluxscan.io/address/cfx:acd97pbhxm11cegrz3anuuvcuj9j0nh956a108f5cc?tab=contract-viewer 的checkMintPermission来确认自己的领取权限。目前传入crowdin-LouisL.eth 应为大于0的数，代表具有领取权限

**Reply by LouisL19851** [reply]
已经领取完成了，但是领取完成后页面显示的哈希点击后跳转的链接错误，正确的是https://evm.confluxscan.io/tx/0x8a32bfb509c035e556c0b3d63bea0183af63e995aa0eabfb1d7dfd7e62222cc6
页面显示的是https://testnet.confluxscan.io/transaction/0xdff392e99ad27a119c8c3b30dfd084a166458e8c30ee9c40e48624714538d6d5
没啥大问题
![image](https://github.com/Conflux-Chain/conflux-documentation/assets/101488673/8b52e0fd-53a2-4ce6-bce8-ce52b080c8df)

**Reply by darwintree** [reply]
感谢反馈。实际上应该是
https://www.confluxscan.io/transaction/0xdff392e99ad27a119c8c3b30dfd084a166458e8c30ee9c40e48624714538d6d5
espace的交易是这条交易的子交易。相关配置项可能出了问题，我们这边检查一下
